src/Core/Content/Category/Service/CategoryBreadcrumbBuilder.php

```diff
@@ -31,7 +31,6 @@
 use Shopware\Storefront\Framework\Seo\SeoUrlRoute\NavigationPageSeoUrlRoute;
 
 /**
- * @experimental stableVersion:v6.7.0 feature:BREADCRUMB_STORE_API
  * related methods: getProductBreadcrumbUrls, loadProduct, getCategoryForProduct, loadCategory,
```

Comment:
the whole comment can be removed. as it was an additional information for the experimental annotation